### PR TITLE
Core: Camera: only accept ViewEvent if getBoundingBox_ is set.

### PR DESCRIPTION
**Pull request by r-englund**

### CODE REVIEW

**Review by martinfalk — APPROVED** [review]

File: apps/inviwo/CMakeLists.txt [hunk 1]
```diff
@@ -173,3 +173,12 @@ endif()
 # Save information for python tools.
 ivw_mod_name_to_class(ivw_module_classes ${enabled_modules})
 ivw_private_create_pyconfig("${IVW_MODULE_DIR};${IVW_EXTERNAL_MODULES}" "${ivw_module_classes}" inviwo)
+
+
+# Windows specific build steps
```
Comment (petersteneteg) [inline apps/inviwo/CMakeLists.txt:178]: What is this? Why is it here?

File: resources/changelog.html [hunk 2]
```diff
@@ -1,821 +0,0 @@
...
-	.ne { color: #a6e22e } /* Name.Exception */
-	.nf { color: #a6e22e } /* Name.Function */
-	.nl { color: #f8f8f2 } /* Name.Label */
-	.nn { color: #f8f8f2 } /* Name.Namespace */
-	.nx { color: #a6e22e } /* Name.Other */
-	.py { color: #f8f8f2 } /* Name.Property */
-	.nt { color: #f92672 } /* Name.Tag */
-	.nv { color: #f8f8f2 } /* Name.Variable */
-	.ow { color: #f92672 } /* Operator.Word */
-	.w { color: #f8f8f2 } /* Text.Whitespace */
```
Comment (petersteneteg) [inline resources/changelog.html:59]: Why did you delete the changelog?

**Review by petersteneteg — CHANGES_REQUESTED** [review]

File: apps/inviwo/CMakeLists.txt (hunk 1 above)
Comment (r-englund) [reply apps/inviwo/CMakeLists.txt:178]: My bad, was testing some stuff, must have accidentally added it to the commit

File: resources/changelog.html (hunk 2 above)
Comment (r-englund) [reply resources/changelog.html:59]: It is always marked as changed on my computers, guess I staged it by mistake

**Comment by r-englund** [comment]
rebased, new PR #690

File: resources/changelog.html (hunk 2 above)
Comment (martinfalk) [reply resources/changelog.html:59]: That is most likely causes by an older version of either markdown or gfm python modules.
